application/src/main/java/io/redlink/smarti/services/ImporterService.java

```diff
@@ -0,0 +1,150 @@
+package io.redlink.smarti.services;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Iterator;
+
+import org.apache.http.client.ClientProtocolException;
+import org.bson.types.ObjectId;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.redlink.smarti.services.importer.I_ConversationSource;
+import io.redlink.smarti.services.importer.I_ConversationTarget;
+import io.redlink.smarti.webservice.ConversationWebservice;
+import io.redlink.smarti.webservice.RocketChatEndpoint;
+import io.redlink.smarti.webservice.pojo.RocketEvent;
+
+/**
+ * This service provides an Smarti importer.<p>
+ * 
+ * Sources for import data to Smarti are:
+ * <li>Directly from Rocket.Chat MongoDB</li>
+ * <li>A Rocket.Chat MongoDB Export</li>
+ * <li>An E-Mailbox</li>
+ * <li>A FAQ website</li>  
+ * 
+ * @author Ruediger Kurz
+ */
+@Service
+public class ImporterService implements I_ImporterService {
+	
+	private Logger log = LoggerFactory.getLogger(ImporterService.class);
+
+	/** A simple ISO date formatter. */
+	private static final DateFormat ISODateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+
+    @Autowired
+    private ConversationWebservice conversationWebservice;
+    
+    @Autowired
+    private RocketChatEndpoint rcEndpoint;
+    
+    public ImporterService() {
+    		rcEndpoint = new RocketChatEndpoint(null, 0, null);
+    }
+	
+    long time;
+    
+	/**
+	 * @see I_ImporterService#importComversation(I_ConversationSource, I_ConversationTarget)
+	 */
+	@Override
+	public void importComversation(I_ConversationSource source, I_ConversationTarget target) throws Exception {
+
+		
+		if (source != null && target != null) {
+			time = new Date().getTime();
+			log.info("Start export at    : " + (new Date().getTime() - time) + " ms");
+			String export = source.exportConversations();
+			log.info("Start parsing at   : " + (new Date().getTime() - time) + " ms");
+			JSONObject eportedJSON = (JSONObject) new JSONParser().parse(export);
```

Comment:
The idea was to avoid duplicated code, since
`JSONObject eportedJSON = (JSONObject) new JSONParser().parse(export);`
would be the same in each Source implementation.

Could you please provide some arguments why you'll expect the Source itself should create the Object?